experimental/packages/opentelemetry-shim-opencensus/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenCensus shim
```

Comment:
Maybe a disclaimer that this package is not yet released so people don't find the readme and get confused when the install command fails